pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -726,6 +726,15 @@ stdenvNoCC.mkDerivation {
       echo "-isystem ${getDev libcxx}/include/c++/v1" >> $out/nix-support/libcxx-cxxflags
       echo "-stdlib=libc++" >> $out/nix-support/libcxx-ldflags
     ''
+    # GCC NG friendly libc++
+    + optionalString (libcxx != null && libcxx.isGNU or false) ''
+      for dir in ${getDev libcxx}/include/c++/*; do
+        echo "-isystem $dir" >> $out/nix-support/libcxx-cxxflags
+      done
+      for dir in ${getDev libcxx}/include/c++/*/${targetPlatform.config}; do
```

Comment:
I'm hoping we don't need this because we're not creating that directory with GCC NG. Can we remove this?

Reply:
No, I get missing headers that are located in a bits directory when I didn't have it. Already tried it.